ext/nnstreamer/tensor_filter/tensor_filter_openvino.cc

```diff
@@ -363,20 +437,70 @@ TensorFilterOpenvino::getOutputTensorDim (GstTensorsInfo * info)
   return ret;
 }
 
+/**
+ * @brief Do inference using Inference Engine of the OpenVino framework
+ * @param prop property of tensor_filter instance
+ * @param[in] input the array of input tensors
+ * @param[out] output the array of output tensors
+ * @return RetSuccess if OK. non-zero if error
+ */
+int
+TensorFilterOpenvino::invoke (const GstTensorFilterProperties * prop,
+    const GstTensorMemory * input, GstTensorMemory * output)
+{
+  InferenceEngine::BlobMap inBlobMap;
+  InferenceEngine::BlobMap outBlobMap;
+  guint num_tensors;
+  guint i;
+
+  num_tensors = (prop->input_meta).num_tensors;
+  for (i = 0; i < num_tensors; ++i) {
+    const GstTensorInfo *info = &((prop->input_meta).info[i]);
+    InferenceEngine::Blob::Ptr blob = convertGstTensorMemoryToBlobPtr (
+        this->_inputTensorDescs[i], &(input[i]));
```

Comment:
Is omitting `.data` from `input[i]` is intentional? Both works, but wanted to know what is a good practice.

Reply:
@kparichay  Yes, it is intentional. In the case of a class that has a standard layout (i.e., C struct), it is legal. It had been regarded as a performance optimization technique (related to the elimination of unnecessary memory access blah blah, I am not sure). However, I think this is a bad habit right now. I will change it. Thanks.

Reply:
@kparichay In the previous comment, there is a misunderstanding about the code you pointed out. 
```
InferenceEngine::Blob::Ptr TensorFilterOpenvino::convertGstTensorMemoryToBlobPtr (
    const InferenceEngine::TensorDesc tensorDesc, const GstTensorMemory * gstTensor)
```
This is a static function taking ``` const GstTensorMemory * gstTensor```.

Reply:
aah, you are right
I thought it was directly passing the GstTensorMemory to make the blob